package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "private": true,
+  "scripts": {
+    "test": "npm run test-unit-local",
+    "test-unit-local": "karma start tests/karma.conf.local.js",
+    "test-unit-saucelabs": "karma start tests/karma.conf.saucelabs.js"
+  },
+  "engines": {
+    "node": ">=10"
+  },
+  "devDependencies": {
+    "http-server": "^0.12.3",
+    "karma": "5.2.3",
+    "karma-chrome-launcher": "3.1.0",
+    "karma-firefox-launcher": "2.1.0",
+    "karma-qunit": "4.1.1",
+    "karma-requirejs": "1.1.0",
+    "karma-sauce-launcher": "^4.3.3",
+    "nightwatch": "^1.5.1",
+    "qunit": "2.12.0",
+    "requirejs": "2.3.3"
```

Comment:
Would it cause any issues that we have an npm requirejs, and also (in index.html) a version loaded via script tags?

Reply:
The npm requirejs is used by the QUnit runner. I didn't want to switch over to it for the non-test use case because that means you would have to run `npm install` as part of releases as well, which would be a larger change and also mean the repo no longer works plainly after a checkout from Git. You may also want to have some kind of workflow practice written down for that if we go that route, with regards to how and which packages we fetch from npm. npm offers a lock file, but it isn't particularly meaningful since it generally gets re-written every time and people don't generally write down and compare signatures before/after.

My recommendation would be to only use npm for dev tooling so that you never have to commit or release its artefacts, and can run it in isolated containers locally and in CI throw-away contexts. Once it starts crossing that line into releases, you'd want a bit more thought into it I think.

I'll see if I can make it so that the checked-in copy of requirejs for releases continues to be used from the repo as well.

Reply:
@Krinkle I'm very happy to follow your recommendation. You're clearly much more experienced with npm-enabled Repos than I am.